CHANGELOG.rst

```diff
@@ -20,6 +20,9 @@ number of the code change for that issue.  These PRs can be viewed at:
 
 3.7.1 (unreleased)
 ==================
+
+- Added python 3.12 to testing matrix for Jenkins and github actions. [#1843]
```

Comment:
Please move the python 3.12 comment out of section 3.7.1 and put it in a new section 3.7.2.